src/core/control/Control.cpp

```diff
@@ -1976,9 +1979,27 @@ void Control::toolFillChanged() {
 }
 
 void Control::toolLineStyleChanged() {
+    if (toolHandler->getLineStyle().getClearLineStyle()) {
```

Comment:
```suggestion
    if (!toolHandler->hasCapability(TOOL_CAP_LINE_STYLE)) {
```

Reply:
Resolved by fea96d645547dc7db4ef2605aa2ad8c802c8b945 and 660d3f958c816c90568a3594831aa74ada84761c.